.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "image": "mcr.microsoft.com/vscode/devcontainers/base:ubuntu",
+  "features": {
+    "ghcr.io/devcontainers/features/go:1": {
+      "version": "1.21"
+    },
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {}
+  },
+  "postCreateCommand": ".devcontainer/postCreateCommand.sh",
+  "workspaceMount": "source=${localWorkspaceFolder},target=/home/vscode/go/src/github.com/cnoe-io/idpbuilder,type=bind",
+  "workspaceFolder": "/home/vscode/go/src/github.com/cnoe-io/idpbuilder",
```

Comment:
Does this have to be under the default GOPATH? I would prefer to have this directly under the user's home path. e.g. `/home/vscode/workspaces/idpbuilder`

Reply:
@nabuskey please take another look when you have a chance, I simplify the setup to the bare minimal now, I think